220-ofcom-22000.md

```diff
@@ -1,4 +1,58 @@
 # 2.20. OFCOM \(-22000\)
+## ofcom\_odbc\: SQLConnect failed
 
+* Description
+  * This error occurs when ODBC is failed to run.
+* Action 
+  * Check if tibero is running and boot if it is not.
+  
+  ---
+  
+  ### Case 1: ODBC failed
+  
+  * Problem 
+    * dataset migration fails with below message
+    ```
+    ofcom_odbc: SQLConnect failed
+    - State: ERRCL
+    - Native Error: -2131
+    - Error Message: [unixODBC] Generic I/O error.
+    012504 E MIG0201E ams_initialize() failed. - rc = -22110
+    ```
+  * Solution
+    1. Check if tibero is running, by using ps -eLf | grep tibero 
+    ```bash
+    jh_lee@jh_lee:~$ ps -eLf | grep tibero
+    jh_lee    2637  2334  2637  0    1 02:56 pts/0    00:00:00 grep --color=auto tibero
+    ```
+    2. If you see just one line, it means it is not running. So you have to make it down and then boot it by using:
+    ```
+    jh_lee@jh_lee:~$ tbdown clean
+    jh_lee@jh_lee:~$ tbboot
+    ```
+    3. Now, if you try to use migrate the dataset, it will give the below message:
+    ```
+    (E) CLI3003 unable to connect to main server : 127.0.0.1 [CLI0106] [Connection refused]
+    (E) CLI0100 getenv error : TMAX_BACKUP_ADDR [CLI0102]
+    013528 E MIG0222E lockm_lock_all() failed. - rc = -1
+    013528 E MIG0603E Failed to import - rc = -1
+    ```
+    So in this case, you type:
+    ```
+    jh_lee@jh_lee:~$ tmboot
+    jh_lee@jh_lee:~$ tjesmgr boot
+    
+    ```
+    After that, you should be able to migrate without any issues
+    
+---
 
```

Comment:
Move below contents to 215-cpmlib-15000.md, since the error code is 15010.
Also, change the title to follow the title name convention.